scss/_variables.scss

```diff
@@ -1077,6 +1077,9 @@ $form-floating-input-padding-b:         .625rem !default;
 $form-floating-label-height:            1.5em !default;
 $form-floating-label-opacity:           .65 !default;
 $form-floating-label-transform:         scale(.85) translateY(-.5rem) translateX(.15rem) !default;
```

Comment:
This line is probably no longer needed.

Reply:
Should I remove this line and create a new PR then?